### PR TITLE
US impact improvements

### PR DESCRIPTION
**Pull request by nikhilwoodruff**
Fixes a bug causing changes on a no-reform, adds abolition switches and blank-slate parameter switches.

### CODE REVIEW

File: policyengine/countries/us/reform_parameters.yaml [hunk 1]
```diff
@@ -10,6 +10,116 @@ reforms:
         variable: income_tax
         tests:
           - increases_net_income: true
+    abolish_emp_payroll_tax:
```
Comment (MaxGhenis) [inline policyengine/countries/us/reform_parameters.yaml:13]: More verbose but I think it'd be clearer to exactly match the `abolish_` switch names to the variable they abolish.
```suggestion
    abolish_employee_payroll_tax:
```

File: policyengine-client/src/countries/us/us.jsx [hunk 2]
```diff
@@ -200,6 +221,16 @@ export class US extends Country {
                 ],
             }
         },
+        "HUD": {
```
Comment (MaxGhenis) [inline policyengine-client/src/countries/us/us.jsx:224]: Could you add the US flag to HUD and HHS?

**Review by MaxGhenis — APPROVED** [review]
Just tried it out--this is great.

File: policyengine/countries/us/reform_parameters.yaml (hunk 1 above)
Comment (MaxGhenis) [reply policyengine/countries/us/reform_parameters.yaml]: I tried to fix this and it didn't work, even adding to `reform_parameters.yaml` as well. So leaving as-is for now.

**Comment by MaxGhenis** [comment]
Went ahead and added more logos and organization to the policy parameter menu.
image
image

image

**Comment by nikhilwoodruff** [comment]
Awesome, thanks.

**Pull request merged**